common/sgx/verifier.c

```diff
@@ -614,14 +783,20 @@ static oe_result_t _get_verifier_plugins(
 {
     oe_result_t result = OE_UNEXPECTED;
     size_t uuid_count = 0;
+    const oe_uuid_t* uuids[4];
 
     if (!verifiers || !verifiers_length)
         OE_RAISE(OE_INVALID_PARAMETER);
 
+    uuids[0] = &_ecdsa_uuid;
+    uuids[1] = &_ecdsa_report_uuid;
+    uuids[2] = &_ecdsa_quote_uuid;
+
 #ifdef OE_BUILD_ENCLAVE
-    uuid_count = 2; // In enclave, only support local and ECDSA formats
+    uuids[3] = &_local_uuid;
+    uuid_count = 4; // In enclave, only support local and 3 ECDSA formats
 #else
-    uuid_count = 1; // In host, only support ECDSA format
+    uuid_count = 3; // In host, only support 3 ECDSA formats
```

Comment:
Just a suggestion... it would be nice to have a // comment explaining why local isn't supported in host.

Reply:
Thanks!